.gitignore

```diff
@@ -4,5 +4,6 @@
 .coverage
 .mypy_cache/
 .pytest_cache/
+.venv/
```

Comment:
Also assuming you actually want this as a directory, and not a file as mentioned in the PR title